.github/workflows/ci.yaml

```diff
@@ -33,23 +33,26 @@ jobs:
         if: matrix.platform == 'ubuntu-latest'
         run: |
           sudo apt update
-          sudo apt install ffmpeg gobject-introspection libgirepository1.0-dev pandoc
-          poetry install --with=release --extras=docs --extras=replaygain --extras=reflink
-          poe docs
+          sudo apt install ffmpeg gobject-introspection libcairo2-dev libgirepository1.0-dev pandoc
```

Comment:
I accidentally also tested this while working on my lastgenre PR, and can confirm that installing libcairo2-dev fixed a pygobjects error.